platforms/core-execution/build-cache-http/src/integTest/groovy/org/gradle/caching/http/internal/HttpBuildCacheServiceIntegrationTest.groovy

```diff
@@ -274,7 +274,7 @@ class HttpBuildCacheServiceIntegrationTest extends HttpBuildCacheFixture {
 
         then:
         withBuildCache().fails("jar")
-            .assertHasCause("Using insecure protocols with remote build cache, without explicit opt-in, is unsupported.")
+            .assertHasDescription("Using insecure protocols with remote build cache, without explicit opt-in, is unsupported.")
```

Comment:
This is because the "could not start your build" context exception is not longer created.